macaroonbakery/httpbakery/_client.py

```diff
@@ -132,6 +133,13 @@ def acquire_discharge(self, cav, payload):
         if resp.status_code == 200:
             return bakery.Macaroon.from_dict(resp.json().get('Macaroon'))
         cause = Error.from_dict(resp.json())
```

Comment:
Is this line a leftover?
Assuming this raises a JSONDecodeError, how is it possible that the test is passing?

Reply:
yeah, sorry, I was making sure my unit test worked and then failed to remove it